tests/src/test/java/alluxio/shell/command/PersistCommandTest.java

```diff
@@ -67,6 +67,56 @@ public void persistDirectoryTest() throws IOException, AlluxioException {
   }
 
   @Test
+  public void persistMultiFilesTest() throws IOException, AlluxioException {
+    String filePath1 = "/testPersist/testFile1";
+    String filePath2 = "/testFile2";
+    String filePath3 = "/testPersist/testFile3";
+    FileSystemTestUtils.createByteFile(mFileSystem, filePath1, WriteType.MUST_CACHE, 10);
+    FileSystemTestUtils.createByteFile(mFileSystem, filePath2, WriteType.MUST_CACHE, 20);
+    FileSystemTestUtils.createByteFile(mFileSystem, filePath3, WriteType.MUST_CACHE, 30);
+
+    Assert.assertFalse(mFileSystem.getStatus(new AlluxioURI(filePath1)).isPersisted());
+    Assert.assertFalse(mFileSystem.getStatus(new AlluxioURI(filePath2)).isPersisted());
+    Assert.assertFalse(mFileSystem.getStatus(new AlluxioURI(filePath3)).isPersisted());
+
+    int ret = mFsShell.run("persist", filePath1, filePath2, filePath3);
+    Assert.assertEquals(0, ret);
+    checkFilePersisted(new AlluxioURI(filePath1), 10);
+    checkFilePersisted(new AlluxioURI(filePath2), 20);
+    checkFilePersisted(new AlluxioURI(filePath3), 30);
+  }
+
+  /**
+   * Tests persisting several files and directories together in one persist command.
+   *
+   * @throws IOException
+   * @throws AlluxioException
+   */
+  @Test
+  public void persistMultiFilesAndDirsTest() throws IOException, AlluxioException {
+    ClientContext.getConf().set(Constants.USER_FILE_WRITE_TYPE_DEFAULT, "MUST_CACHE");
+    AlluxioShellUtilsTest.resetFileHierarchy(mFileSystem);
```

Comment:
Run `ClientContext.resetClientContext()` after resetting the file hierarchy so that this test doesn't interfere with other tests.